docs/setup/connect-to-elasticsearch.asciidoc

```diff
@@ -1,53 +1,54 @@
 [[connect-to-elasticsearch]]
 == Add your data
 ++++
-<titleabbrev>Add data</titleabbrev>
+<titleabbrev>Add an integration</titleabbrev>
 ++++
 
-To start working with your data in {kib}, use one of the many ingest options,
-available from the home page.
-You can collect data from an app or service,
-or upload a file that contains your data.  If you're not ready to use your own data,
-add a sample data set and give {kib} a test drive.
+The best way to add data is to use one of our many integrations.
+Alternatively, you can add a sample data set or upload a file.
```

Comment:
In product, both sample data and file upload are listed as integrations. As are enterprise search ingestion options (upload a JSON file, etc.), elasticsearch language clients, etc. I agree it sounds a bit odd, but at this point should we group them into this category?